portal/collections/_foundations/accessibility.html

```diff
@@ -449,7 +450,7 @@ <h3 class="title fw-bold" id="focus-indicator">Focus indicators</h3>
     <hr />
     <p class="my-5">A focus indicator serves as a visual signal that highlights the element currently in focus on a web page or application. This feature is essential for users who depend on keyboard navigation or assistive technologies to engage with digital content.</p>
     
-    <div class="d-grid gap-4" style="grid-template-columns: repeat(auto-fill, minmax(17rem, 1fr));">
+    <div class="d-grid gap-4" style="grid-template-columns: repeat(auto-fit, minmax(15rem, 1fr));">
```

Comment:
would you want to try bootstrap 5 new css-grid over writing inline style? https://getbootstrap.com/docs/5.1/layout/css-grid/

I have $enabled-cssgrid in custom.style.scss. I tried it when using the cards implementation in component page and it helps to customise the size of the columns. u can see if i will work for this case.

Reply:
Thanks for the info! this is very useful, I have updated it according to the implementation.